src/axom/mir/examples/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+# other Axom Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+add_subdirectory(concentric_circles)
+add_subdirectory(tutorial_simple)
```

Comment:
When the Elvira algorithm is added, do you expect to extend these examples to support EquiZ and Elvira?

I assume so, but if not, perhaps these examples should be renamed w/ EquiZ.

Reply:
That makes sense.